plugin-server/tests/helpers/clickhouse.ts

```diff
@@ -30,23 +31,23 @@ export async function resetTestDatabaseClickhouse(extraServerConfig: Partial<Plu
 }
 
 export async function delayUntilEventIngested(
-    fetchEvents: () => Promise<any[] | any>,
+    fetchEvents: () => Promise<Record<string, any>[] | number>,
```

Comment:
I don't think this type is correct - e.g. `fetchSessionRecordingEvents` returns an array not a object/record and it's being used widely.

Reply:
I see I misread the type. Using `Record<string, any>` as a 'this is a row from the database' is confusing imo. Why not any[] or use a generic type argument?